### PR TITLE
Fix for method overloading

### PR DESCRIPTION
**Pull request by kartikeytewari-ul**
This fixes selective logging for methods with overloading of arguments. Now each method has a distinct counter. 
- Associated Ticket: #83 

### Compile Pipeline
- run from CI pipeline

### Replay Pipeline
1. USMD
- SDK v0.6.2
usmd-0 6 2

- SDK v0.6.102
usmd-0 6 102

2. Webflux Demo
- SDK v0.6.2
webflux-0 6 2

- SDK v0.6.102
webflux-0 6 102

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -10,7 +10,7 @@
 
     <artifactId>unlogged-sdk</artifactId>
     <groupId>video.bug</groupId>
-    <version>0.6.3</version>
+    <version>0.6.102</version>
```
Comment (artpar) [inline pom.xml:13]: remove this, you can keep this change local to yourself

File: src/main/java/io/unlogged/util/MapStoreUtil.java [hunk 2]
```diff
@@ -2,10 +2,15 @@
 
 import io.unlogged.Constants;
 
-public class MapStoreName {
+public class MapStoreUtil {
```
Comment (artpar) [inline src/main/java/io/unlogged/util/MapStoreUtil.java:5]: this needs a better name since it has a very specific usage

File: pom.xml (hunk 1 above)
Comment (kartikeytewari-ul) [reply pom.xml]: okay using version `0.6.3`.

File: src/main/java/io/unlogged/util/MapStoreUtil.java (hunk 2 above)
Comment (kartikeytewari-ul) [reply src/main/java/io/unlogged/util/MapStoreUtil.java]: refactored the name here

**Pull request merged**